wye/profiles/forms.py

```diff
@@ -10,24 +11,20 @@ class UserAuthenticationForm(AuthenticationForm):
                                widget=forms.PasswordInput,
                                required=True)
 
-
-class SignupForm(UserCreationForm):
-    '''
-    '''
-    username = forms.CharField(label=_("Email"), max_length=100, required=True)
-    password1 = forms.CharField(label=_("Password"), widget=forms.PasswordInput,
-                                required=True)
-    password2 = forms.CharField(label=_("Password confirmation"),
-                                widget=forms.PasswordInput,
-                                help_text=_("Enter the same password as above, for verification."))
-    firstname = forms.CharField(label=_("First Name"), max_length=150,
-                                required=True)
-    lastname = forms.CharField(label=_("Last Name"), max_length=150,
-                               required=True)
-
+class SignupForm(forms.ModelForm):
     mobile = forms.CharField(label=_("Mobile"), max_length=10,
-                             required=True)
-
+                                required=True,
+                                widget=forms.TextInput(attrs={'placeholder': 'Mobile Number'}))
+    class Meta:
+        model = get_user_model()
+        fields = ['first_name', 'last_name']
+        widgets = {
+            'first_name': forms.TextInput(attrs={'placeholder': 'First Name'}),
+            'last_name': forms.TextInput(attrs={'placeholder': 'Last Name'}),
+        }
+    def save(self, user):
+        user.profile.mobile = self.cleaned_data['mobile']
```

Comment:
Well, I forgot to tell that, after assigning **user.profile.mobile** **user.profile.save()** will save the actual profile modifications. You can find any other way to save the profile record update. Thanks!